docs/resources/project.md

```diff
@@ -0,0 +1,36 @@
+# Project Resource
+
+The top-level concept in Codefresh. You can create projects to group pipelines that are related. In most cases a single project will be a single application (that itself contains many micro-services). You are free to use projects as you see fit. For example, you could create a project for a specific Kubernetes cluster or a specific team/department.
+
```

Comment:
Please consider to add:
https://codefresh.io/docs/docs/configure-ci-cd-pipeline/pipelines/#pipeline-concepts

Reply:
according to the [docs[(https://www.terraform.io/docs/registry/providers/docs.html#resource-data-source-headers) the description should contain a few words about what resource does and links to the official documentation.
Adde link to the docs.